nipoppy_cli/nipoppy/data/examples/sample_global_config-all_pipelines.json

```diff
@@ -0,0 +1,144 @@
+{
+    "DATASET_NAME": "<DATASET_NAME>",
+    "VISITS": [
+        "<VISIT_LABEL>"
+    ],
+    "SESSIONS": [
+        "ses-<SESSION_LABEL>"
+    ],
+    "CONTAINER_CONFIG": {
+        "ARGS": [
+            "--cleanenv"
+        ]
+    },
```

Comment:
Is there a reason / time when `--cleanenv` is not needed when using containers? If having this flag will never hurt, then might as well hardcode it pipeline class and save space here!